tests/system_tests/test_functional/test_util/test_util.py

```diff
@@ -0,0 +1,16 @@
+import pathlib
+import subprocess
+
+import pytest
+
+
+@pytest.mark.wandb_core_only(reason="does not depend on service")
+def test_util_import_adds_attribute_to_parent_module():
+    script = pathlib.Path(__file__).parent / "util_import_lazy.py"
+    proc = subprocess.Popen(
```

Comment:
[check_call](https://docs.python.org/3/library/subprocess.html#subprocess.check_call) is what you're looking for. You don't need to set `stdout` (and setting it to PIPE without reading from it can cause a deadlock)